src/utils/py/daos_api.py

```diff
@@ -1876,13 +1876,15 @@ def create(self, coh, epoch):
     #  snapshots and the epochs and names lists. See description of
     #  daos_cont_list_snap in src/include/daos_api.h. This must be done for
     #  DAOS-1336 Verify container snapshot info.
-    def list(self, coh):
+    def list(self, coh, epoch=None):
         """ Call daos_cont_snap_list and make sure there is a snapshot in the
         list.
         coh --ctype.u_long handle on an open container
         Returns the value of the epoch for this DaosSnapshot object.
         """
         func = self.context.get_function('list-snap')
+        if epoch is None:
+            epoch = self.epoch
         num = ctypes.c_uint64(1)
         epoch = ctypes.c_uint64(self.epoch)
         anchor = Anchor()
```

Comment:
I think we should just report back a dictionary with the list of epochs that have been snapshotted.